README.md

```diff
@@ -66,6 +66,9 @@ The file `deploy/kubei.yaml` is used to deploy and configure Kubei on your clust
 
 ![](images/kubei-results.png)     
 
+## Running Kubei behind Proxy
+
+Uncomment proxy env variables configuration under Clair and Kubei deployments in `deploy/kubei.yaml`.
```

Comment:
Uncomment and configure the proxy env variables for the Clair and Kubei deployments in `deploy/kubei.yaml`.